src/libs/actions/User.js

```diff
@@ -552,9 +552,22 @@ function subscribeToUserEvents() {
         if (_.isArray(pushJSON)) {
             updates = pushJSON;
         } else {
-            // const lastUpdateID = pushJSON.lastUpdateID;
-            // const previousUpdateID = pushJSON.previousUpdateID;
             updates = pushJSON.updates;
+
+            // Not always we'll have the lastUpdateID and previousUpdateID properties in the pusher update
+            // until we finish the migration to reliable updates. So let's check it before actually updating
+            // the properties in Onyx
+            if ('lastUpdateID' in pushJSON && 'previousUpdateID' in pushJSON) {
```

Comment:
NAB
```suggestion
            if (!_.isUndefined(pushJSON.lastUpdateID) && !_.isUndefined(pushJSON.previousUpdateID)) {
```
Or I think this is even better if there's no 0 value for ID:
```suggestion
            if (pushJSON.lastUpdateID && pushJSON.previousUpdateID) {
```

Reply:
Changed usaged.